channels/candidate-4.4.yaml

```diff
@@ -17,3 +17,6 @@ versions:
 - 4.4.0-rc.6
 - 4.4.0-rc.7
 - 4.4.0-rc.8
+
+# 4.4.0-rc.9 Errata: https://access.redhat.com/errata/RHBA-2020:0581
```

Comment:
I don't think we need this, since it can be extracted from the release image itself.

Reply:
Dropped with f4190e5 -> 963de21.

/lgtm